rfc6962-bis.xml

```diff
@@ -381,13 +381,13 @@ REDACT(label) = prefix || BASE32(index || LABELHASH(keyid || label))</artwork>
             <spanx style="verb">BASE32</spanx> is the Base 32 Encoding function (section 6 of <xref target="RFC4648"/>). Pad characters MUST NOT be appended to the encoded data.
           </t>
           <t>
-            <spanx style="verb">index</spanx> is the 1 byte index of a hash function in <xref target="hash_algorithms"/>.
+            <spanx style="verb">index</spanx> is the 1 byte index of a hash function in <xref target="hash_algorithms"/>. The value 255 is reserved.
           </t>
           <t>
             <spanx style="verb">LABELHASH</spanx> is the hash function identified by <spanx style="verb">index</spanx>.
           </t>
           <t>
-            <spanx style="verb">keyid</spanx> is the keyIdentifier from the Subject Key Identifier extension (section 4.2.1.2 of <xref target="RFC5280"/>), excluding the ASN.1 OCTET STRING tag and length bytes.
+            <spanx style="verb">keyid</spanx> is the keyIdentifier from the Subject Key Identifier extension (section 4.2.1.2 of <xref target="RFC5280"/>), including the ASN.1 OCTET STRING tag and length bytes.
```

Comment:
The length of the label itself is not specified. What do you think about a simple length prefix - 1 octet for specifying the length of the SKI (excluding the ASN.1 OCTET STRING tag and length bytes) and 1 octet for specifying the length of the label?

So HASH(SKI_LENGTH || SKI || LABEL_LENGTH || LABEL)
?